lib/allora/allora.ts

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env node
+import 'dotenv/config';
+import 'source-map-support/register';
+import * as cdk from 'aws-cdk-lib';
+import { AlloraStack } from './lib/allora-stack';
+
```

Comment:
Use `cdk-nag` to enforce security best practices. See [here](https://github.com/aws-samples/aws-blockchain-node-runners/blob/main/lib/theta/lib/common-stack.ts#L62) for example